Source/Atomic/UI/UIWidget.h

```diff
@@ -250,10 +258,18 @@ class UIWidget : public Object, public tb::TBWidgetDelegate
     bool GetCaptured();
 
     void SetCapturing(bool capturing);
```

Comment:
`GetCapturing`/`SetCapturing` it's getters and setters for `needCapturing_` variable (https://github.com/AtomicGameEngine/AtomicGameEngine/blob/0fdfd80624754be1f1652cc804896a33b86cc7dc/Source/ThirdParty/TurboBadger/tb_widgets.h#L988)
`needCapturing` means if widget can be captured(it is related to the multitouch support, if we have captured widget, then nothing else can be pressed and only that event can recieve mouse events, e.g. in the DPad buttons don't need capturing, because I want have its pressed at the same time, all TB widget need capturing by default)
`GetCaptured` returns if widget is captured